app/scripts/modules/core/src/managed/overview/CollapsibleSection.tsx

```diff
@@ -0,0 +1,29 @@
+import cx from 'classnames';
+import React from 'react';
+
+import { Icon } from '@spinnaker/presentation';
+
+import './CollapsibleSection.less';
+
+interface Props {
+  expandedByDefault?: boolean;
+  title: React.ReactNode;
+}
+
+export const CollapsibleSection: React.FC<Props> = ({ expandedByDefault, title, children }) => {
```

Comment:
there is a component in src/presentation called CollapsibleSection 
consider consolidating these (I assume they do the same thing)
If not, please think of a better name for this one